src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -2,7 +2,9 @@
 
 import java.io.File;
 import java.io.FileNotFoundException;
+import java.io.InputStream;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.InputStream'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)